### ISSUE TITLE
Highlight tab feature

### ISSUE THREAD

**Issue by M0zart** [yfdyh000/tabutils#25]
Add "Highlight tab" popup menu command that makes a tab easily findable visually, when it is one of the many tabs and stacks. May be add settings to configure the highlighted tab view, like for active, unloaded, unread tabs. May be make it wider than other tabs with other font or somehow in other way.

(I'm not sure, but it seems that it's better NOT to change tab height for highlighted tabs, since it decreases performance with multiline tabbar)

**Comment by yfdyh000** [comment]
Do you mean TU Option - Appearance - Tabs? It was already there.
And, the Appearance - Menus tab are also have many useful menus.

**Comment by M0zart** [comment]
No, not exactly. I mean other thing. I usually have lots of tabs in a multiline tab bar open. I want to easily find some of them I consider important - to make them visually different from other tabs. I want to be able to mark a tab as important (or highlighted) - so that this tab has different appearance from other non-important tabs, and be more prominent.

May be the appearance of such a marked tab should be configurable as it is on the TU Option - Appearance - Tabs for other kinds of tabs. 

By the way - before it was possible to change the appearance of a blocked or protected tab (I'm not sure of a correct word, because I use other language in the interface), and use this for highlighting, but now I see much less kinds of tabs in TU Option - Appearance - Tabs. Before I used Tab Utilities Phoenix.

I know about pinned tabs, but I don't want to lose vertical space for a whole row of tabs with only 1-3 tabs.

Offtopic: your extension seems to work perfectly on the latest Palemoon browser.

**Comment by yfdyh000** [comment]
Looks seems the [Tab Utilities Phoenix](https://addons.mozilla.org/firefox/addon/tab-utilities-phoenix/) added some Tabs appearance options for protected and locked, no more.

I have no energy to add tabs coloring/highlighting options, I suggest you look for another extension use with the extension. Like the https://addons.mozilla.org/firefox/addon/tab-colors/ or https://addons.mozilla.org/firefox/addon/tab-kit-tab-highlighter/.

Sounds like a good news if I don't need to make some changes.
